src/js/structs/UniversePackage.js

```diff
@@ -71,14 +45,6 @@ class UniversePackage extends Item {
     return StringUtil.punctuate(this.get("preInstallNotes"));
   }
 
-  getPostInstallNotes() {
```

Comment:
@nLight 	we're currently NOT using the `postInstallNotes` provided by universe-packages. e.g. https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/C/concord/0/package.json#L20.

do you think it's worth investigating whether we should actually use those?